helm/values.yaml

```diff
@@ -1156,11 +1166,21 @@ gateway:
 #    appProtocol: http
   # annotations:
   autoscaling:
+    # When gateway.autoscaling.enabled is true, a HorizontalPodAutoscaler is created
     enabled: true
     minReplicas: 1
     maxReplicas: 3
+    # warning: these two target utilization will be overwritten if gateway.autoscaling.metrics is defined.
     targetAverageUtilization: 50
     targetMemoryAverageUtilization: 80
+    # here you can add specific annotations to this HPA
+    annotations:
```

Comment:
should it be commented?

Reply:
not an issue but yes, missed that one: fixed ✔